.gitignore

```diff
@@ -12,4 +12,6 @@ GeneratedPluginRegistrant.m
 
 GeneratedPluginRegistrant.java
 
-.dart_tool
\ No newline at end of file
+.dart_tool
+packages/image_picker/.vscode/launch.json
+packages/image_picker/.vscode/settings.json
```

Comment:
We don't want plugin-specific lines in the repo-level `.gitignore`. Please replace these two lines by
```gitignore
.vscode/
```